Resources/config/consumers.yml

```diff
@@ -17,3 +19,19 @@ services:
           - [ setSmartesbHelper, ['@smartesb.helper']]
           - [ setEventDispatcher, ['@event_dispatcher']]
           - [ setSerializer, ['@jms_serializer']]
+  smartesb.consumers.php_amqp_lib_async_queue:
```

Comment:
Is still needed this class? I don't see is used in the code

Reply:
The goal to left this class is just in the case that as an Open Source lib, maybe the user don't wants to use the signal functions. So, after all it's just one more option to the lib.

Reply:
Anyway, I will remove this class.